app/views/password_resets/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide :title, t("reset_pass") %>
+<h1><%= t("reset_pass") %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: password_reset_path(params[:id]) do |f| %>
+      <%= render "shared/error_messages", user: @user %>
+
+      <%= hidden_field_tag :email, @user.email %>
```

Comment:
cách diễn đạt của c chưa rõ lắm. c tham khảo 
https://viblo.asia/p/form-tag-va-su-khac-biet-so-voi-form-for-bWrZnLXw5xw.